lib/utils.ts

```diff
@@ -395,6 +409,47 @@ async function requestAsync(
 		opts.headers = nativeHeaders;
 	}
 
+	const { FormData: NodeFormData } =
+		// eslint-disable-next-line @typescript-eslint/no-var-requires
+		require('formdata-node') as typeof import('formdata-node');
+	const { FormDataEncoder } =
+		// eslint-disable-next-line @typescript-eslint/no-var-requires
+		require('form-data-encoder') as typeof import('form-data-encoder');
```

Comment:
Is this going to work in a browser context?

Reply:
Yes, the formdata-node library is Isomorphic but only re-exports native FromData object for browsers.